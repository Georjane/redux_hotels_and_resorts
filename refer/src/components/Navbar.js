import Dashboard from './Dashboard';

function Navbar() {
  return (
    <div className="Navbar">
      <h1>Navbar</h1>
      <Dashboard />
    </div>
  );
}

export default Navbar;
