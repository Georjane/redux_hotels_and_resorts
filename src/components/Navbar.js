const Navbar = () => (
  <nav className="navbar navbar-expand-lg bglight">
    <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo03" aria-controls="navbarTogglerDemo03" aria-expanded="false" aria-label="Toggle navigation">
      <span className="navbar-toggler-icon" />
    </button>
    <a className="navbar-brand white" href="/register">Hotels & Resorts</a>

    <div className="collapse navbar-collapse" id="navbarTogglerDemo03">
      <ul className="navbar-nav mr-auto mt-2 mt-lg-0">
        <li className="nav-item active">
          <a className="nav-link white" href="/register">
            Residences
            <span className="sr-only">(current)</span>
          </a>
        </li>
        <li className="nav-item">
          <a className="nav-link white" href="/register">Dinning</a>
        </li>
        <li className="nav-item">
          <a className="nav-link white" href="/register" tabIndex="-1" aria-disabled="true">Events</a>
        </li>
      </ul>
      <form className="form-inline my-2 my-lg-0">
        <input className="form-control mr-sm-2 transparent" type="search" placeholder="Search" aria-label="Search" />
      </form>
    </div>
  </nav>
);

export default Navbar;
