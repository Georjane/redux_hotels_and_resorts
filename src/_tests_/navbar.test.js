import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

it('renders the navbar properly', () => {
  const tree = renderer.create(<Navbar />);
  expect(tree).toMatchSnapshot();
});
