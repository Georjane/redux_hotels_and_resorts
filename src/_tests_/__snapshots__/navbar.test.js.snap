// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the navbar properly 1`] = `
<nav
  className="navbar navbar-expand-lg bglight fixed"
>
  <button
    aria-controls="navbarTogglerDemo03"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#navbarTogglerDemo03"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <a
    className="navbar-brand white logo"
    href="/register"
  >
    <img
      alt="carosel"
      className="w-25"
      src="images/logo.png"
    />
    <p>
      Address
      <br />
      Hotels & Resorts
    </p>
  </a>
  <div
    className="collapse navbar-collapse"
    id="navbarTogglerDemo03"
  >
    <ul
      className="navbar-nav mr-auto mt-2 mt-lg-0"
    >
      <li
        className="nav-item active"
      >
        <a
          className="nav-link white"
          href="/register"
        >
          Residences
          <span
            className="sr-only"
          >
            (current)
          </span>
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link white"
          href="/register"
        >
          Dinning
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          aria-disabled="true"
          className="nav-link white"
          href="/register"
          tabIndex="-1"
        >
          Events
        </a>
      </li>
    </ul>
    <div>
      <a
        href="login"
      >
        <button
          className="sc-bdfBQB cqoxcE"
        >
          Sign In
        </button>
      </a>
    </div>
    <form
      className="form-inline my-2 my-lg-0"
    >
      <input
        aria-label="Search"
        className="form-control mr-sm-2 transparent"
        placeholder="Search"
        type="search"
      />
    </form>
  </div>
</nav>
`;
